content/en/docs/reference/config-api/kubeadm-config.v1beta2.md

```diff
@@ -721,7 +721,7 @@ for, so other administrators can know its purpose.</td>
     
   
 <tr><td><code>expires</code> <B>[Required]</B><br/>
-<code>invalid type</code>
+<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#time-v1-meta"><code>meta/v1.Time</code></a>
```

Comment:
ideally we should link from API version Foo to API version Foo references.
but there are no 1.22 API docs yet,

Reply:
it doesn't matter much for the kubeadm usage of the meta / core types, since these do not change much, but this could be a problem elsewhere. we could at least link to N-1 (1.21).

Reply:
See comment below.

Reply:
`v1.20` does stand out as not being right here.